airbyte-integrations/connectors/source-twilio/source_twilio/spec.json

```diff
@@ -4,30 +4,63 @@
     "$schema": "http://json-schema.org/draft-07/schema#",
     "title": "Twilio Spec",
     "type": "object",
-    "required": ["account_sid", "auth_token", "start_date"],
+    "required": [
+      "account_sid",
+      "auth_token",
+      "start_date"
+    ],
     "additionalProperties": false,
     "properties": {
       "account_sid": {
         "title": "Account ID",
         "description": "Twilio account SID",
         "airbyte_secret": true,
-        "type": "string"
+        "type": "string",
+        "order": 1
       },
       "auth_token": {
         "title": "Auth Token",
         "description": "Twilio Auth Token.",
         "airbyte_secret": true,
-        "type": "string"
+        "type": "string",
+        "order": 2
       },
       "start_date": {
         "title": "Replication Start Date",
         "description": "UTC date and time in the format 2020-10-01T00:00:00Z. Any data before this date will not be replicated.",
         "pattern": "^[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}Z$",
-        "examples": ["2020-10-01T00:00:00Z"],
-        "type": "string"
+        "examples": [
+          "2020-10-01T00:00:00Z"
+        ],
+        "type": "string",
+        "order": 3
+      },
+      "lookback": {
+        "title": "Lookback",
+        "description": "format in minutes. Any data between sync time and sync time minus lookback setting in minutes will be replicated.",
+        "examples": [
+          "60"
+        ],
+        "default": 0,
+        "type": "string",
+        "order": 4
+      },
+      "page_size": {
```

Comment:
Hello @alafanechere , In our opinion too, but since it was set to the minimum possible and it is a variable that the api allows us to modify, we thought of putting it as an attribute that can be modified by the user, if you think it is not correct we can leave it at the maximum allowed as you suggest. Cheers

Reply:
@alejandroRosano-muttdata yes please use the maximum value and do not expose the parameter to the user. Thanks!